userspace/libsinsp/examples/test.cpp

```diff
@@ -38,254 +48,347 @@ static sinsp_evt_formatter* default_formatter = nullptr;
 static sinsp_evt_formatter* process_formatter = nullptr;
```

Comment:
For even more simplicity would just one default formatter work? e.g. `#define PROCESS_DEFAULTS "*%evt.num %evt.time %evt.cpu %proc.name %thread.tid %evt.dir %evt.type %evt.args %container.id %proc.ppid %proc.pid %proc.exe %proc.cmdline %fd.name %fd.num %fd.sockfamily %fd.l4proto %fd.type %user.uid %user.name %user.loginuid"` for all events (null keys wouldn't be printed anyways) and simply add another arg to pass a custom output string?  @Molter73 also concerned about breaking your new e2e tests, what do you think?

Reply:
As long as we don't remove fields, the e2e tests ignore fields that are not explicitly tested for, we would need to go ahead and manually add the new fields into the tests for them to be taken into account (not ideal, I know, but friendlier than totally failing IMO).